crates/store/re_types/Cargo.toml

```
@@ -79,6 +79,7 @@ once_cell.workspace = true
 ply-rs.workspace = true
 smallvec.workspace = true
 thiserror.workspace = true
+tiff = { version = "0.9.1" }
```

Define it in the main workspace manifest, and then
```suggestion
tiff.workspace = true